### ISSUE TITLE
nmatrix.h not getting found

### ISSUE THREAD

**Issue by thisMagpie** [thisMagpie/fftw#11]
This does not seem to stand in the way of running specs so I am not sure what the implications are but in any case check the [travis compiler issue](https://travis-ci.org/thisMagpie/fftw/jobs/27166204#L135) for details.

**Comment by cjfuller** [comment]
Yep, you don't need nmatrix.h to run/use nmatrix.  Since you've got the nmatrix gem in the dependencies in the gemspec, it's using the installed nmatrix gem for running the specs you've brought over from nmatrix for testing.

If you were to try to run specs where nmatrix-fftw.so was required, they would fail because the build failed and so there is not nmatrix-fftw there to require before the specs.

So to summarize, the issue is that the build of nmatrix-fftw is failing due to not finding the header, but since the specs are currently only testing nmatrix, they still succeed.

**Comment by thisMagpie** [comment]
Ok I think we can close this considering what got decided in https://github.com/thisMagpie/fftw/issues/19
